manifest/com.Sinduy/QuestPro4Resonite/info.json

```diff
@@ -4,6 +4,7 @@
 	"description": "With Quest Pro, you can eye tracking and facial tracking whih ALXR also You need read the instruction in the github page to install the mod.",
 	"category": "Hardware Integration",
 	"sourceLocation": "https://github.com/sjsanjsrh/QuestPro4Resonite",
```

Comment:
May want to archive the repo and/or add a notice to the readme about it not being needed.

Reply:
I've added it to the readme. Thanks for the advice.